css/src/alerts.scss

```diff
@@ -69,20 +69,27 @@ h3 .dashicons-warning {
 .restore .dashicons,
 .dismiss .dashicons {
 	font-size: 24px;
+	width: 24px;
+	height: 24px;
 }
 
 .yoast-bottom-spacing {
 	margin-bottom: 20px;
 }
 
-.restore,
-.dismiss {
-	width: 32px;
+.yoast-alerts .button.restore,
+.yoast-alerts .button.dismiss {
+	flex: 0 0 40px;
+	width: 40px;
 	height: 40px;
-	margin-left: 8px;
-	border: none;
-	background-color: transparent;
-	cursor: pointer;
+	line-height: inherit;
+}
+
+.yoast-alerts .button.restore:hover,
```

Comment:
I prefer to introduce as little nested selectors as possible. My guideline for writing CSS is [CSSBEM](http://getbem.com/introduction/). So we could solve this with either: `.button--yoast-alerts` or `.yoast-alerts__button`.